deepr/jobs/export_xla_model_metadata.py

```diff
@@ -0,0 +1,100 @@
+"""Export xla compatible model metadata from a saved model"""
+
+from dataclasses import dataclass
+import logging
+from typing import List, Dict
+
+import tensorflow as tf
+
+from deepr.io import Path
+from deepr.jobs import base
+from deepr.utils import import_graph_def
+from deepr.jobs._xla_generated.tf2xla_pb2 import Config as XlaMetaConfig
+
+LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class ExportXlaModelMetadata(base.Job):
+    """Export xla compatible model metadata from a saved model
+
+    Attributes
+    ----------
+    path_saved_model : str
+        Path to directory containing SavedModel exports to convert
+    path_metadata : str
+        Path to directory that will contain the metadata
+    graph_name : str
+        Name of the saved model graph (name of the protobuf file)
+    metadata_name : str
+        Name of the metadata file
+    feed_shapes : Dict[str, List[int]]
+        Shapes of feeds to expose
+    fetch_shapes : Dict[str, List[int]]
+        Shapes of fetches to expose
+    """
+
+    path_saved_model: str
+    path_metadata: str
+    graph_name: str
+    metadata_name: str
+    feed_shapes: Dict[str, List[int]]
+    fetch_shapes: Dict[str, List[int]]
+
+    def run(self):
+        # Create session and import graph under scope "model"
+        session = tf.Session(graph=tf.Graph())
+        with session.graph.as_default():
+            import_graph_def(f"{self.path_saved_model}/{self.graph_name}", name="")
+
+            feed_nodes = {
+                "/".join(name.split("/")): node
```

Comment:
all names start with "/" because you used `name = ""` above?
is there a reason why you don't use something more explicit like `model`?

Reply:
adding a prefix here will add a prefix in all the node names, and make it more difficult to find the nodes

however I noticed this code is more complicated than necessary, simplified it